src/components/CopyableText.stories.tsx

```diff
@@ -46,16 +30,18 @@ export default {
       options: ['undefined', 'custom'],
     },
   },
-  component: Component,
-  decorators: [withRouter],
-  title: 'Standalone/CopyableText',
 };
```

Comment:
```suggestion
} satifies Meta<typeof Component>;
```
https://github.com/SEEK-Jobs/ie-ad-selection-panel/blob/master/src/panels/example/Example.stories.tsx#L11